src/app/hierarchical-grid/hierarchical-grid-excel-style-filtering-style/hierarchical-grid-excel-style-filtering-style.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit, ViewChild } from "@angular/core";
+import { IgxHierarchicalGridComponent } from "igniteui-angular";
+import { SINGERS } from "../data";
+
+@Component({
+    selector: "hierarchical-grid-excel-style-filtering-style",
+    styleUrls: ["./hierarchical-grid-excel-style-filtering-style.component.scss"],
+    templateUrl: "hierarchical-grid-excel-style-filtering-style.component.html"
+})
+export class HGridExcelStyleFilteringStyleComponent implements OnInit {
+
+    @ViewChild("hierarchicalGrid", { static: true })
+    public hierarchicalGrid: IgxHierarchicalGridComponent;
+
+    public localdata;
+
+    constructor() {
+        this.localdata = SINGERS;
+    }
+
+    public ngOnInit(): void {
+    }
+
+    public formatter = (a) => a;
```

Comment:
This formatted is not necessary, I believe we can remove it from sample.

Reply:
Addressed.